unittests/launch_box/test_launch_box.cu

```diff
@@ -1,11 +1,19 @@
+#include <iostream>
+
 #include <gunrock/cuda/launch_box.hxx>
 
 using namespace gunrock::cuda;
 
 int main(void) {
   typedef launch_box_t<
-    ampere<16, 64, 0>,
-    turing<16, 32, 0>,
-    kepler<32, 64, 0>
+    ampere<16, 64, 2>,
```

Comment:
Does this support `ampere<sm_80<>, ...>` yet?

Reply:
Not yet, was wanting to make sure we got everything else settled before I move onto that.